### PR TITLE
Document loadmoduleid in plugin description

### PR DESCRIPTION
**Pull request by Quy**
Pull Request for Issue #25074.

### Summary of Changes
Add `loadmoduleid` syntax in plugin description.

### Testing Instructions
Code review
or
* Go to settings of plugin `Content - Load Modules`.
* Read the description.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_content_loadmodule.ini [hunk 1]
```diff
@@ -11,4 +11,4 @@ PLG_LOADMODULE_FIELD_VALUE_HORIZONTAL="Wrapped by table (horizontal)"
 PLG_LOADMODULE_FIELD_VALUE_MULTIPLEDIVS="Wrapped by Multiple Divs"
 PLG_LOADMODULE_FIELD_VALUE_RAW="No wrapping (raw output)"
 PLG_LOADMODULE_FIELD_VALUE_TABLE="Wrapped by table (column)"
-PLG_LOADMODULE_XML_DESCRIPTION="Within content this plugin loads Module positions, Syntax: {loadposition user1} or Modules by name, Syntax: {loadmodule mod_login}. Optionally can specify module style and for loadmodule a specific module by title, Syntax: {loadmodule mod_login,module title,style}."
+PLG_LOADMODULE_XML_DESCRIPTION="Within content this plugin loads Modules by ID, Syntax: {loadmoduleid 1} or Module positions, Syntax: {loadposition user1} or Modules by name, Syntax: {loadmodule mod_login}. Optionally can specify module style and for loadmodule a specific module by title, Syntax: {loadmodule mod_login,module title,style}."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_content_loadmodule.ini]: PLG_LOADMODULE_XML_DESCRIPTION="Within content this plugin loads a Module by ID, Syntax: {loadmoduleid 1} or a Module by position, Syntax: {loadposition user1} or a Module by name, Syntax: {loadmodule mod_login}. Optionally can specify module style and for loadmodule a specific module by title, Syntax: {loadmodule mod_login,module title,style}."

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 4098f7af8f86baf3eecc160aa2120ba27c358c53<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25076">issues.joomla.org/tracker/joomla-cms/25076</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 4098f7af8f86baf3eecc160aa2120ba27c358c53<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25076">issues.joomla.org/tracker/joomla-cms/25076</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25076">issues.joomla.org/tracker/joomla-cms/25076</a>.</sub>

**Comment by ReLater** [comment]
> Optionally can specify module style

moduleid can't. See https://github.com/joomla/joomla-cms/pull/25072
That's why I had problems to write a correct description yesterday on my own.

**Comment by Quy** [comment]
Will this work? `Optionally can specify module style (except for loadmoduleid)`

**Comment by HLeithner** [comment]
thx

**Pull request merged**